fwupdate/source/cc32xx/build.sh

```diff
@@ -0,0 +1,148 @@
+#
+# Copyright 2014-2015 CyberVision, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#!/bin/bash
+
+RUN_DIR=`pwd`
+
+function help {
+    echo "Choose one of the following: {build|run|deploy|clean}"
+    exit 1
+}
+
+if [ $# -eq 0 ]
+then
+    help
+fi
+
+APP_NAME="demo_client"
+PROJECT_HOME=$(pwd)
+BUILD_DIR="build"
+LIBS_PATH="libs"
+KAA_LIB_PATH="$LIBS_PATH/kaa"
+KAA_C_LIB_HEADER_PATH="$KAA_LIB_PATH/src"
+KAA_CPP_LIB_HEADER_PATH="$KAA_LIB_PATH/kaa"
+KAA_SDK_TAR="kaa-client*.tar.gz"
+
+#Wifi settings
+SSID="xxx"
```

Comment:
Maybe leave this empty and check in build script that this settings are specified?